docs/site/components/docs/docs.js

```diff
@@ -149,6 +149,8 @@ angular
                 }),
               description: $sce.trustAsHtml(
                   formatHtml(detectLinks(detectModules(obj.description.full)))),
+              lineNum: obj.codeStart,
+              lineNumLink: obj.codeStart ? '#L' + obj.codeStart : '',
```

Comment:
Yeah it was a little daunting at first but luckily there's enough code around that I could kinda just copy what already existed and re-shape for my needs.